subprojects/gst-plugins-bad/sys/msdk/gstmsdkvpp.c

```diff
@@ -498,12 +551,10 @@ gst_msdkvpp_create_buffer_pool (GstMsdkVPP * thiz, GstPadDirection direction,
   pool = gst_msdk_create_va_pool (&info, thiz->context, use_dmabuf,
       min_num_buffers);
 #else
-  /* Currently use system pool for windows path */
+  pool = gst_msdk_create_d3d11_pool (thiz, &info, min_num_buffers);
+#endif
   if (!thiz->use_video_memory)
```

Comment:
We always set _thiz->use_video_memory_ as TRUE in gst_msdkvpp_set_caps, anywhere or under any condition this value can be FALSE?